src/pizzeria/Order.java

```diff
@@ -0,0 +1,57 @@
+package pizzeria;
+
+import java.time.LocalTime;
+import java.util.*;
+
+public class Order {
+    private final int orderNumber;
+    private final int customerNumber;
+    private final List<Integer> quantityOfOrder;
+    private final LocalTime time = LocalTime.now();
+
+    Order(int customerNumber, List<Integer> quantityOfOrder) {
+        Random order = new Random(System.currentTimeMillis());
+        this.orderNumber = order.nextInt(1000) + 10000;
+        this.customerNumber = customerNumber;
+        this.quantityOfOrder = quantityOfOrder;
+        for (int i = 0; i < this.quantityOfOrder.size(); i++) {
+            if (this.quantityOfOrder.get(i) > 10) {
+                System.err.println("You can't order more than 10 pizzas at one time");
+                this.quantityOfOrder.set(i, 10);
+            }
+        }
+    }
+
+    Order() {
+        Random order = new Random(System.currentTimeMillis());
+        this.orderNumber = order.nextInt(1000) + 10000;
+        this.customerNumber = getCustomerNumber();
```

Comment:
**getCustomerNumber(); and getQuantityOfOrder();** methods returns **0 and null**